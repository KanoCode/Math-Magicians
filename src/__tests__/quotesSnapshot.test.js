import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Quotes from '../Components/pages/quote';

it('renders correctly', () => {
  const tree = renderer
    .create(<BrowserRouter>
      <Quotes />
    </BrowserRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
