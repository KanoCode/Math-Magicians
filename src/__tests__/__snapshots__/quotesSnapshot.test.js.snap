// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section>
  <p>
    “Mathematics expresses values that reflect the cosmos, including orderliness, balance, harmony, logic, and abstract beauty.”
     
    <strong>
      ― Deepak Chopra....
    </strong>
  </p>
</section>
`;
